pkgs/development/tools/google-app-engine/python-sdk/default.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs, stdenv, python27Full, python27Packages, makeWrapper }:
+with {
+  python = python27Full;
```

Comment:
As far as I know, we only need `sqlite3`. Is there a way to construct a python that only has this module? I see the `includeModules` boolean, but I don't see an arg to specify a particular set of modules.